### ISSUE TITLE
Hi, I have a few questions

### ISSUE THREAD

**Issue by uwkwi** [antoinehabis/Deep-ContourFlow#1]
Hello, why is the result of my operation different from that of your paper? As shown in the figure, why is the outline of the lion's body detected instead of the outline of the lion's face detected? I am looking forward to your answer
<img width="299" alt="loin" src="https://github.com/user-attachments/assets/8fa0dfa8-45e2-49d3-affa-16dbfeae23b1">

**Comment by antoinehabis** [comment]
Hello Yijinzou,

Since the publication of the paper, I made a few changes in the
backpropagation like adding a smart gradient descent using adam optimizer.
If you clone the repository like it is, you should have access to rather
good hyperparameters that help you have the right segmentation result.
Basically two hyperparameters can cause this kind of result:

- the learning rate
- the area force that controls the constraint on the area of the contour.

I hope I answered your questions and i you have more don't hesitate,

Antoine
